ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.18__arm64-nightly.yaml

```diff
@@ -829,6 +829,50 @@ tests:
     test:
     - chain: openshift-e2e-test-qe
     workflow: cucushift-installer-rehearse-baremetalds-ipi-ovn
+- as: baremetal-compact-agent-ipv4-static-connected-f7-compliance
+  cluster: build05
+  cron: 19 16 6,13,20,27 * *
+  steps:
+    cluster_profile: equinix-ocp-metal-qe
+    dependencies:
+      OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE: release:arm64-latest
+    env:
+      AUX_HOST: openshift-qe-metal-ci.arm.eng.rdu2.redhat.com
+      RESERVE_BOOTSTRAP: "false"
+      architecture: arm64
+      ipv4_enabled: "false"
+      ipv6_enabled: "true"
+      masters: "3"
+      workers: "0"
+    test:
+    - ref: openshift-extended-test
+    - ref: openshift-extended-web-tests
+    - ref: openshift-e2e-test-qe-report
+    workflow: baremetal-lab-agent-install-network-static
+- as: baremetal-compact-agent-ipv4-static-connected-f7-compliance-destructive
```

Comment:
This is a workflow without default sc created, all the test cases will be skipped on it.